packages/react/src/components/header/components/headerActionBar/HeaderActionBarNavigationMenu.tsx

```diff
@@ -504,6 +486,14 @@ export const HeaderActionBarNavigationMenu = ({
       id="hds-mobile-menu"
       onTransitionEnd={animationDone}
     >
+      {!!actionBarItems &&
```

Comment:
Hmm, this whole block could be replaced with a lot shorter

`{actionBarItems?.map?.((item: HeaderActionBarItemProps) => {
     return React?.cloneElement?.(item as unknown as React.ReactElement, { fullWidth: true, key: uuidv4() });
})}`

Works in the storybook example at least, ofc might break somehow 🤷